app/presenters/schools/courses/curriculum_presenter.rb

```diff
@@ -70,6 +71,12 @@ def targets
           }
         end
       end
+
+      def vimeo_access_token?
+        token = @course.school.configuration['vimeo_access_token'] || 
+          Rails.application.secrets.vimeo_access_token
+        token.present? ? true : false
```

Comment:
There's no need for the ternary here.

```suggestion
        token.present?
```